graphium/data/datamodule.py

```diff
@@ -1286,8 +1291,8 @@ def _make_multitask_dataset(
             progress=self.featurization_progress,
             about=about,
             save_smiles_and_ids=save_smiles_and_ids,
-            data_path=self._path_to_load_from_file(stage) if load_from_file else None,
-            load_from_file=load_from_file,
+            data_path=self._path_to_load_from_file(stage) if processed_graph_data_path else None,
+            processed_graph_data_path=processed_graph_data_path,
```

Comment:
```suggestion
            dataloading_from=self.dataloading_from
```
We don't need `processed_graph_data_path` here but rather `data_path` as it is otherwise hardcoded to "ram" in the MultitaskDataset